xds/src/main/java/io/grpc/xds/XdsLoadBalancer.java

```diff
@@ -360,6 +340,38 @@ void updateFallbackServers(
       }
     }
 
+    private void handleFallbackAddressesOrNameResolutionError() {
```

Comment:
Why does the method name contain "NameResolutionError"? It's only called from fallback-related methods. What this method does is passing the latest fallback addresses to the fallback balancer. The handling of empty addresses is just part of it. Maybe "propagateFallbackAddresses()".

Reply:
Renamed